package.json

```diff
@@ -28,9 +28,11 @@
   "homepage": "https://github.com/bitfinexcom/hypercore-logs#readme",
   "dependencies": {
     "@hyperswarm/replicator": "2.0.2",
+    "chokidar": "3.5.1",
     "debug": "4.3.2",
     "glob": "7.2.0",
     "hypercore": "9.12.0",
+    "is-glob": "4.0.3",
```

Comment:
I think we already have this functionality locally https://github.com/bitfinexcom/hypercore-logs/blob/master/src/helper.js#L31, could we reuse the local one and omit the lib?